jvector-base/src/main/java/io/github/jbellis/jvector/graph/disk/feature/FeatureId.java

```diff
@@ -32,13 +30,11 @@
  */
 public enum FeatureId {
     INLINE_VECTORS(InlineVectors::load),
-    FUSED_ADC(FusedADC::load),
+    FUSED_PQ(FusedPQ::load),
```

Comment:
There is a small, probably zero, chance that this feature was used in a previous version of CC/HCD. Because we are repurposing this enum position, I propose that we fail the load if the graphs version is less than 6 so we can fail predictably and the message will be to recreate the graph.

Reply:
IIUC, the fused adc feature was available before on disk format 6. I am proposing that when we load the features from disk, if we have fused pq and version < 6, we throw an exception.